stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/channel/info/ChannelInfo.kt

```diff
@@ -122,50 +120,66 @@ public fun ChannelInfo(
+                onClick = { onChannelOptionClick(Cancel) }
+            )
     ) {
-        Column(
-            modifier = Modifier.padding(top = 16.dp),
-            verticalArrangement = Arrangement.Center,
-            horizontalAlignment = Alignment.CenterHorizontally
+        Card(
+            modifier
+                .clickable(
+                    indication = null,
+                    interactionSource = MutableInteractionSource(),
+                    onClick = { }
+                ),
+            elevation = 8.dp,
+            shape = shape,
+            backgroundColor = ChatTheme.colors.barsBackground,
         ) {
+            Column(
+                modifier = Modifier.padding(top = 16.dp),
+                verticalArrangement = Arrangement.Center,
+                horizontalAlignment = Alignment.CenterHorizontally
+            ) {
 
-            Text(
-                modifier = Modifier.padding(horizontal = 16.dp),
-                text = title,
-                style = ChatTheme.typography.bodyBold,
-                color = ChatTheme.colors.textHighEmphasis,
-                maxLines = 1,
-                overflow = TextOverflow.Ellipsis
-            )
-            Text(
-                text = stringResource(
-                    id = R.string.stream_compose_channel_members,
-                    channelMembers.size,
-                    onlineMembers
-                ),
-                style = ChatTheme.typography.footnoteBold,
-                color = ChatTheme.colors.textLowEmphasis,
-            )
+                Text(
+                    modifier = Modifier.padding(horizontal = 16.dp),
+                    text = title,
+                    style = ChatTheme.typography.bodyBold,
+                    color = ChatTheme.colors.textHighEmphasis,
+                    maxLines = 1,
+                    overflow = TextOverflow.Ellipsis
+                )
+                Text(
+                    text = stringResource(
+                        id = R.string.stream_compose_channel_members,
+                        channelMembers.size,
+                        onlineMembers
+                    ),
+                    style = ChatTheme.typography.footnoteBold,
+                    color = ChatTheme.colors.textLowEmphasis,
+                )
 
-            LazyRow(
-                modifier = Modifier
-                    .fillMaxWidth()
-                    .padding(top = 8.dp, bottom = 16.dp)
-                    .align(Alignment.CenterHorizontally),
-                horizontalArrangement = Arrangement.spacedBy(12.dp),
-                contentPadding = PaddingValues(start = 16.dp, end = 16.dp)
-            ) {
-                items(selectedChannel.members) { member ->
-                    ChannelInfoUserItem(member = member)
+                LazyRow(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(top = 8.dp, bottom = 16.dp)
```

Comment:
So `vertical` applies padding both on `top` and `bottom`, and it applies the same value - since by design the top padding is a bit smaller than the bottom one, we could use `vertical` and add a `Spacer`, but it seemed easier to me to just use two different values.

Does this make sense to you?

Reply:
Yes